cypress/e2e/supervision/contacts/edit-contact.cy.js

```diff
@@ -0,0 +1,35 @@
+beforeEach(() => {
+  cy.loginAs("Case Manager");
+  cy.createClient()
+    .withContact();
+  cy.get("@client").then(({id}) => {
+    cy.visit(
+      `/supervision/#/clients/${id}`
+    );
+  });
+  cy.get(".TABS_CONTACTS button").click();
+});
+
+describe(
+  "Edit contact",
+  {tags: ["@supervision-core", "@contact", "@smoke-journey"]},
+  () => {
+    it("can edit a non organisation contact", () => {
+      cy.get(".contacts-list").contains("a", "Edit").click();
+      cy.get("footer").contains("button", "Save & update contact").as("editContactButton");
+
+      cy.get("input[name=firstName]").type("A".repeat(256));
+      cy.get("@editContactButton").click();
+      cy.get(".validation-summary")
+        .and("contain.text", "There is a problem")
+        .and("contain.text", "First name - The input is more than 255 characters long");
+
+      cy.get("input[name=firstName]").clear().type("Edited");
```

Comment:
```suggestion
      cy.get("input[name=firstName]").clear();
      cy.get("input[name=firstName]").type("Edited");
```

Reply:
we've had problems in the past with these two being chained. Docs say it's unsafe to chain actions together

Reply:
Good spot